var/spack/repos/builtin/packages/py-h5py/package.py

```diff
@@ -11,12 +11,12 @@ class PyH5py(PythonPackage):
     HDF5 library from Python."""
 
     homepage = "http://www.h5py.org/"
-    url      = "https://pypi.io/packages/source/h/h5py/h5py-2.10.0.tar.gz"
+    pypi     = "h5py/h5py-3.1.0.tar.gz"
```

Comment:
```suggestion
    pypi     = "h5py/h5py-3.1.0.tar.gz"
    url      = "https://pypi.io/packages/source/h/h5py/h5py-2.10.0.tar.gz"
```
This `pypi` is only on the newest Spack and didn't make it into the release we're using. Adding the URL should be fine, will be easy to merge next time around :)